src/plugins/media/nodeviews/mediaNodeView/media.tsx

```diff
@@ -148,6 +165,20 @@ export class MediaNode extends Component<MediaNodeProps, MediaNodeState> {
     }
   }
 
+  fetchCozyImage = async () => {
+    try {
+      const server = await serviceClient.getImage(this.props.node.attrs.url)
```

Comment:
So we are storing an url in the schéma and not an id?!

Reply:
Storing the id in the url basically, then fetching the real url from the "id url"